config/routes.rb

```diff
@@ -36,4 +37,15 @@
   get "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinary_offices#veterinarians"
   get "/veterinary_offices/:veterinary_office_id/veterinarians/new", to: "veterinarians#new"
   post "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinarians#create"
+
+  get "/applications/new", to: "applications#new"
+  get "/applications/:id", to: "applications#show"
+  post "/applications", to: "applications#create"
+  patch "/applications/:id", to: "applications#update"
+
+  post "/application_pet", to: "application_pet#create"
+
+  get "/admin/shelters", to: "admin#index"
+  get "/admin/applications/:id", to: "admin#show"
+  patch "/admin/applications/:id", to: "admin#update"
```

Comment:
See my comment in the AdminController but here is where you could make use of using an admin directory. i.e.
```
  get "/admin/shelters", to: "admin/shelters#index"
  get "/admin/applications/:id", to: "admin/applications#show"
  patch "/admin/applications/:id", to: "admin/applications#update"
```